apps/dokploy/templates/discord-tickets/docker-compose.yml

```diff
@@ -1,14 +1,13 @@
 version: "3.8"
 
 services:
-  mysql:
+  tickets-postgres:
     image: mysql:8
     restart: unless-stopped
-    hostname: mysql
     networks:
       - dokploy-network
     volumes:
-      - tickets-mysql:/var/lib/mysql
+      - tickets-mysql-data:/var/lib/mysql
```

Comment:
Why is the container called postgres and the image is taken by mysql

Reply:
LMAO @WoWnik good point, can you explain here @DrMxrcy ?

Reply:
Just a mistake in the mass rename, I can sort it out when I get home.